packages/auth/src/access-token/index.ts

```diff
@@ -34,19 +34,8 @@ export class AccessToken {
   private expirationDate?: Date
 
   constructor(parameters: RequireExactlyOne<AccessTokenParameters, 'refreshToken' | 'scopes'>) {
-    const clientId = parameters.clientId ?? process.env.AWS_CLIENT_ID
-    const clientSecret = parameters.clientSecret ?? process.env.AWS_CLIENT_SECRET
-
-    if (!clientId) {
-      throw new AccessTokenError('clientId is required')
-    }
-
-    if (!clientSecret) {
-      throw new AccessTokenError('clientSecret is required')
-    }
-
-    this.clientId = clientId
-    this.clientSecret = clientSecret
+    this.clientId = parameters.clientId
+    this.clientSecret = parameters.clientSecret
```

Comment:
Quick question: should the clientId/clientSecret/refreshToken be private?

Reply:
I put all the properties in public in order to have information about the instance, but we can put them in private if you prefer to